src/components/core/styles/mixins/buttons.scss

```diff
@@ -113,28 +103,11 @@
     @include icon-button-variables;
 
     #{$button-selector} {
-      @include typo.text-xs--bold;
-      @include typo.font-smoothing;
```

Comment:
are those really never needed? but probably yes, as there can only be an icon? but why have they been there before :D

Reply:
A brutal copy-paste :D